src/components/message/styledMessages.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components/macro';
+
+export const Inner = styled.div`
+  display: flex;
+  align-items: flex-end;
+  flex-direction: ${({ direction }) => direction};
+  margin: auto;
+  width: 100%;
+  @media (max-width: 1000px) {
```

Comment:
using media queries scoped to the component rather than nesting them with other styles per window size is very exciting.